zng/cc.rs

```diff
@@ -311,7 +312,9 @@ pub fn build_zlib_ng(target: &str, compat: bool) {
             }
 
             // neon
-            cfg.define("ARM_NEON", None);
+            if features.contains("neon") {
```

Comment:
Even though right now `neon` is unique, I am afraid that in future there will be `neon-ultra` or something like that which might make this a false positive.

Can you try `features.split(",").any(|name| name == "neon")` instead?

Reply:
Sure, fixed with a62591dc3f3f9cce04f7a02459618c5767243a0e